lib/contact_searcher.rb

```diff
@@ -107,6 +105,12 @@ def api_where(selector=nil)
             end
           when 'first_name', 'last_name'
             self.new_selector[k] = v.is_a?(String)? Regexp.new(v,Regexp::IGNORECASE) : v
+          when 'tags'
+            cs = Tag.find(v).map{|t| t.contact_ids}.flatten
+            contacts_with_tags = cs.select{|c| cs.count(c) > 1}.uniq
+            andit({
+              :_id => {'$in' => contacts_with_tags}
+            })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.